x-pack/plugin/runtime-fields/qa/rest/build.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: 'elasticsearch.testclusters'
```

Comment:
We should replace all of this with the `elasticsearch.yaml-rest-test` plugin. Additionally the test suite should move from `src/test` to `src/yamlRestTest` to comply with new conventions.

Reply:
Sure.

Reply:
b392332ced3a2f896b962453b45600c397ef7602